cloud: update release notes (#10347)
pingcap/docs: tidb-cloud/release-notes-2022.md

# TiDB Cloud Release Notes in 2022

This page lists the release notes of [TiDB Cloud](https://pingcap.com/tidb-cloud/?from=en) in 2022.

## September 6, 2022

**General changes**

* Upgrade the default TiDB version of new [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters from [v6.1.0](https://docs.pingcap.com/tidb/stable/release-6.1.0) to [v6.1.1](https://docs.pingcap.com/tidb/stable/release-6.1.1).

**Console changes**

* Now you can [apply for a PoC](/tidb-cloud/tidb-cloud-poc.md) from the entry in the upper-right corner of the TiDB Cloud console.

> **Cloud API changes**

Reviewer: ```suggestion
**API changes**
```

> * Support increasing the storage of TiKV or TiFlash nodes through the [TiDB Cloud API](/tidb-cloud/api-overview.md). You can use the `storage_size_gib` field of the API endpoint to do the scaling.

Reviewer 2: ```suggestion
* Support increasing the storage of a TiKV or TiFlash node through the [TiDB Cloud API](/tidb-cloud/api-overview.md). You can use the `storage_size_gib` field of the API endpoint to do the scaling.
```

This page lists the release notes of [TiDB Cloud](https://en.pingcap.com/tidb-cloud/) in 2022.

## September 6, 2022

**General changes**

* Upgrade the default TiDB version of TiDB Cloud [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters from [v6.1.0](https://docs.pingcap.com/tidb/stable/release-6.1.0) to [v6.1.1](https://docs.pingcap.com/tidb/stable/release-6.1.1).

> * Provide [more metrics](https://docs.pingcap.com/tidbcloud/monitor-datadog-integration#metrics-available-to-datadog) of Dedicated Tier clusters in Datadog to help you better understand the cluster performance status.

Reviewer: ```suggestion
* Provide [more metrics](/tidb-cloud/monitor-datadog-integration.md#metrics-available-to-datadog) of Dedicated Tier clusters in Datadog to help you better understand the cluster performance status.
```

>     If you have [integrated TiDB Cloud with Datadog](https://docs.pingcap.com/tidbcloud/monitor-datadog-integration), you can view these metrics in your Datadog dashboards directly.

Reviewer: ```suggestion
    If you have [integrated TiDB Cloud with Datadog](/tidb-cloud/monitor-datadog-integration.md), you can view these metrics in your Datadog dashboards directly.
```

> This page lists the release notes of [TiDB Cloud](https://en.pingcap.com/tidb-cloud/) in 2022.

Reviewer 2: ```suggestion
This page lists the release notes of [TiDB Cloud](https://pingcap.com/tidb-cloud/?from=en) in 2022.
```

## September 6, 2022

> **General change**
> 
> * Upgrade the default TiDB version of new [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters from [v6.1.0](https://docs.pingcap.com/tidb/stable/release-6.1.0) to [v6.1.1](https://docs.pingcap.com/tidb/stable/release-6.1.1).
> 
> **Console change**
> 
> * Now you can [apply for a PoC](/tidb-cloud/tidb-cloud-poc.md) from the entry in the upper-right corner of the TiDB Cloud console.
> 
> **Cloud API change**

Author: ```suggestion
**General changes**

* Upgrade the default TiDB version of new [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters from [v6.1.0](https://docs.pingcap.com/tidb/stable/release-6.1.0) to [v6.1.1](https://docs.pingcap.com/tidb/stable/release-6.1.1).

**Console changes**

* Now you can [apply for a PoC](/tidb-cloud/tidb-cloud-poc.md) from the entry in the upper-right corner of the TiDB Cloud console.

**Cloud API changes**
```

> * Support increasing the node storage of TiKV or TiFlash through the [TiDB Cloud API](/tidb-cloud/api-overview.md). You can use the `storage_size_gib` field of the API endpoint to do the scaling.

Author: ```suggestion
* Support increasing the storage size of TiKV or TiFlash nodes through the [TiDB Cloud API](/tidb-cloud/api-overview.md). You can use the `storage_size_gib` field of the API endpoint to do the scaling.
```

> * Support increasing the node storage of TiKV or TiFlash through the [TiDB Cloud API](/tidb-cloud/api-overview.md). You can use the `storage_size_gib` field of the API endpoint to do the scaling.

Reviewer 2: ```suggestion
* Support increasing the storage of TiKV or TiFlash nodes through the [TiDB Cloud API](/tidb-cloud/api-overview.md). You can use the `storage_size_gib` field of the API endpoint to do the scaling.
```

> * Support increasing the storage size of TiKV or TiFlash nodes through the [TiDB Cloud API](/tidb-cloud/api-overview.md). You can use the `storage_size_gib` field of the API endpoint to do the scaling.

Author: ```suggestion
* Support increasing the storage of TiKV or TiFlash nodes through the [TiDB Cloud API](/tidb-cloud/api-overview.md). You can use the `storage_size_gib` field of the API endpoint to do the scaling.
```
